tests/middleware/test_session.py

```diff
@@ -125,3 +125,20 @@ def test_invalid_session_cookie(test_client_factory):
     # we expect it to not raise an exception if we provide a bogus session cookie
     response = client.get("/view_session", cookies={"session": "invalid"})
     assert response.json() == {"session": {}}
+
+
+def test_session_cookie(test_client_factory):
+    app = create_app()
+    app.add_middleware(SessionMiddleware, secret_key="example", max_age=None)
+    client = test_client_factory(app)
+
+    response = client.post("/update_session", json={"some": "data"})
+    assert response.json() == {"session": {"some": "data"}}
+
+    # check cookie max-age
+    set_cookie = response.headers["set-cookie"]
+    assert "Max-Age" not in set_cookie
+
+    client.cookies.clear_session_cookies()
```

Comment:
Note to myself: use `client.cookies.clear` instead of `client.cookies.clear_session_cookies` on https://github.com/encode/starlette/pull/1376

Reply:
why? afaik this method will keep all other cookies except session-scoped.

Reply:
Oh right, yeah... `httpx` doesn't have `clear_session_cookies` :disappointed: